src/EnergyPlus/ZoneEquipmentManager.cc

```diff
@@ -727,10 +727,16 @@ namespace ZoneEquipmentManager {
 						Temp = CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolDesTemp;
 						HumRat = CalcZoneSizing( CurOverallSimDay, ControlledZoneNum ).CoolDesHumRat;
 						DeltaTemp = Temp - Node( ZoneNode ).Temp;
+						if ( DataHeatBalance::Zone( ActualZoneNum ).HasApproachTempToReturnAir && !( DataGlobals::BeginSimFlag ) ) {						
+							DeltaTemp = Temp - DataHeatBalance::Zone( ActualZoneNum ).AdjustedTempToReturnAir;
+						}
 						// If the user specify the design cooling supply air temperature difference, then
```

Comment:
@mjwitte HERE.

Reply:
At first glance, these changes appear to be correct.  Do the sizing results for the zone cooling design flow rate change as expected?